test/gtest/ucp/ucp_test.cc

```diff
@@ -97,6 +97,10 @@ void ucp_test::cleanup() {
 void ucp_test::init() {
     test_base::init();
 
+    if (RUNNING_ON_VALGRIND && has_transport("ud_v")) {
```

Comment:
can it accidentially disable those configurations where `ud_v` is just one of the transports?

Reply:
Good catch, thanks, fixed!